src/openforms/formio/rendering/nodes.py

```diff
@@ -143,7 +143,9 @@ def is_visible(self) -> bool:
                 self.component, artificial_repeating_group_data
             ):
                 return False
-        elif not is_visible_in_frontend(self.component, self.step_data):
+        elif self.mode != RenderModes.registration and not is_visible_in_frontend(
```

Comment:
I don't think this is right, because now it falls back to render configuration (either explicit or using a default) which depends on the component type.

Probably a more correct approach is to *always* consider a component visible in registration mode, like the export above, which should be changed to:

```py
        if self.mode in {RenderModes.export, RenderModes.registration}:
            return True
```
we want to bypass `is_visible_in_frontend` entirely during registration, however I'm not sure how this plays with the registration email backend :thinking:

Reply:
I have tested the `email registration`  and I don't see a problem. Do you mean something like showing the hidden field's value for example?

Reply:
Yes, that's what I was a bit worried about! Thanks for testing :)